ctru-rs/src/services/ps.rs

```diff
@@ -30,23 +30,44 @@ pub enum AESKeyType {
     Keyslot39Nfc,
 }
 
-pub fn local_friend_code_seed() -> crate::Result<u64> {
-    let mut seed: u64 = 0;
+pub struct Ps(());
```

Comment:
I'm assuming you decided we don't need the service reference machinery here because of the atomic counter internal to `ps` ?

Reply:
The `ServiceReference` is only useful for services that shouldn’t be initialised more than once at a time (especially in multi-threaded programs). `Ps` is an extremely simple service that just give access to a couple of functions and keeps no internal state. There shouldn’t be much of an issue.